cockpit/leapp.html

```diff
@@ -332,6 +337,11 @@ <h4 class="text-center">Application names already in use:</h4>
         */
         var new_leapp_call = function(command, options) {
             command = [LEAPP_CONF["tool-path"]].concat(command);
+            return new_generic_call(command, options);
+        }
+
+        var new_generic_call = function(command, options) {
+            //command = [LEAPP_CONF["tool-path"]].concat(command);
```

Comment:
please remove this line, we shouldn't keep code which is not used, after that LGTM